core/src/main/java/bio/terra/pearl/core/service/survey/SurveyParseUtils.java

```diff
@@ -110,7 +153,20 @@ public static String unmarshalSurveyQuestionChoices(JsonNode question) {
             if (choice.isTextual()) {
                 choices.add(new QuestionChoice(choice.asText(), choice.asText()));
             } else {
-                choices.add(new QuestionChoice(choice.get("value").asText(), choice.get("text").asText()));
+                JsonNode textNode = choice.get("text");
+
+                String text;
+                if (textNode.isTextual()) {
+                    text = textNode.asText();
+                } else {
+                    if (textNode.has("en")) {
+                        text = textNode.get("en").asText();
+                    } else {
+                        text = textNode.get("value").asText();
+                    }
+                }
+
+                choices.add(new QuestionChoice(choice.get("value").asText(), text));
```

Comment:
So, this is a drive by fix that might need more discussion. If you create a question with choices that have multi-language labels, our survey parsing infrastructure would create it with an empty string for the text, since `.asText()` will return an empty string if you call it on an object. This means in heartdemo, any multi-lang dropdown will actually show up empty on export.

The "correct" way to do this would be to create the question definitions based upon the default language. But the issue is that the default language is defined at the _portal environment_ level whereas surveys are at the _portal_ level.

I figured, survey question choices are only displayed in the admin API, so we should just try to match everything else and grab english. If that doesn't exist, grab the raw value. Do you guys think there's a better way to do this?